rmgpy/rmgobject.pyx

```diff
@@ -1,10 +1,11 @@
-# cython: embedsignature=True, cdivision=True
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
```

Comment:
This actually isn't necessary either, since this is technically not a python file.